.github/workflows/dependabot-sync.yml

```
@@ -0,0 +1,45 @@
+name: dependabot-sync
+on:
+  workflow_call:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: ${{ github.repository }}
+
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ github.repository_owner }}/meta
+          path: ${{ github.repository_owner }}/meta
+
+      - run: |
+          DEPENDABOT_FILE="${{ github.repository_owner }}/meta/dependabot/dependabot-${{ github.event.repository.name }}.yml"
+          if [ ! -f $DEPENDABOT_FILE ]; then
+            DEPENDABOT_FILE="${{ github.repository_owner }}/meta/dependabot/dependabot.yml"
+          fi
```

To keep files short and concise, we can have the repository specific settings stored in `dependabot-repo.yml`, then they get concatenated here. wdyt?
```suggestion
          cat "${{ github.repository_owner }}/meta/dependabot/dependabot.yml" > "${{ github.repository }}/dependabot.yml"
          DEPENDABOT_FILE="${{ github.repository_owner }}/meta/dependabot/dependabot-${{ github.event.repository.name }}.yml"
          if [ -f "$DEPENDABOT_FILE" ]; then
            cat "$DEPENDABOT_FILE" >> "${{ github.repository }}/dependabot.yml"
          fi
```

---

There would be at least three exceptions. These repos don't have any Go code, so they have config for GHA and/or npm only.

- `dependabot/dependabot-nur.yml`
- `dependabot/dependabot-vhs-action.yml`
- `dependabot/dependabot-soft-serve-action.yml`

---

Sure, in that case, since they don't have a `go.mod` file, dependabot will just skip that part, right? If so I think that's fine, especially if it will reduce the overhead and maintenance in the future.